### PR TITLE
R20.1 256

### PR DESCRIPTION
**Pull request by dougkenik**
Fix for #256

### CODE REVIEW

File: SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml [hunk 1]
```diff
@@ -252,6 +259,8 @@ Item {
                 } else {
                     flipIcon.color = UM.Theme.getColor("text_inactive");
                 }
+
+                textLoadDialogMagnitude.text = bcListForces.model.loadMagnitude=
```
Comment (bradymadams) [inline SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml:263]: Is that trailing `=` intentional?

**Review by bradymadams — COMMENTED** [review]
Looks good, just the one syntax thing that I think is a typo.

This doesn't address the other problem I found with crashing when "Cancel" is clicked when the load dialog is not visible, right?

**Comment by dougkenik** [comment]
> Looks good, just the one syntax thing that I think is a typo.
> 
> This doesn't address the other problem I found with crashing when "Cancel" is clicked when the load dialog is not visible, right?

It does not address that

File: SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml [hunk 2]
```diff
@@ -260,7 +260,7 @@ Item {
                     flipIcon.color = UM.Theme.getColor("text_inactive");
                 }
 
-                textLoadDialogMagnitude.text = bcListForces.model.loadMagnitude=
+                textLoadDialogMagnitude.text = bcListForces.model.loadMagnitude
```
Comment (bradymadams) [inline SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml:263]: Semi-colon would be preferred.

**Review by bradymadams — APPROVED** [review]
Th

**Pull request merged**